tedge/src/cli/connect/bridge_config.rs

```diff
@@ -168,6 +176,7 @@ bridge_attempt_unsubscribe false
             clean_session: true,
             notifications: false,
             bridge_attempt_unsubscribe: false,
+            bridge_max_packet_size: "32KB".into(),
```

Comment:
Why here we have 32KB and in other place it is 16KB?

Reply:
fixed